packages/admin-vue3/src/ui/Sidebar.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <TransitionRoot appear :show="open" as="template">
```

Comment:
@jannescb I added the `TransitionRoot` component from tailwinds headlessui. This allows us to add transition childs, so transitions that depend on the transition root. Could you please apply the slide left transition to the component? Here are the associated [docs](https://headlessui.dev/vue/transition).